sourcegraph/sourcegraph
# [SG-22831] - Use `production` build for frontend integration tests

## Description
Support running integration tests on top of the production build of the web application.

## Ref
[Gitstart Ticket](https://clients.gitstart.com/sourcegraph/1/tickets/SG-22831)
[SG Issue](https://github.com/sourcegraph/sourcegraph/issues/22831)

## Test Plan
1. Run the web integration build command: `sg run web-integration-build`
2. Notice form that the production build is being generated.
